vectordatasource/transform.py

```diff
@@ -4471,6 +4471,19 @@ def _guess_network_ua(tags):
     return networks
 
 
+def _guess_network_za(tags):
+    ref = tags.get('ref', '')
+    networks = []
+
+    for part in re.split('[;/,]', ref):
```

Comment:
Think it's worth compiling regular expressions in advance? Or will it just not matter here?